pycode/examples/simulation/oseir_simple.py

```diff
@@ -0,0 +1,79 @@
+#############################################################################
+# Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+#
+# Authors: Martin J. Kuehn, Wadim Koslow, Daniel Abele
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+from memilio.simulation import Damping
+from memilio.simulation.oseir import Model, simulate, Index_InfectionState
+from memilio.simulation.oseir import InfectionState as State
+import numpy as np
+import argparse
+
+
+def run_secir_simulation(show_plot = True):
+    """
+    Runs the c++ secir model using one age group 
+    and plots the results
+    """
```

Comment:
I think this is copy paste from secir. Seir doesn't have age groups, and this example doesn't plot anything.